### DISCUSSION TITLE
Inline interactive code blocks

### DISCUSSION THREAD

**Discussion by samhodgkinson** [squidfunk/mkdocs-material#5286] [General]
### Context

I'm looking to see if its possible to add Inline interactive code blocks, please see the below link to an example on Github. 
https://learn.microsoft.com/en-us/contribute/code-in-docs#interactive-code-snippets

### Description

Where code is provided within a code snippet, users reading the documentation would be able to run the specific code against Azure using Azure Cloud Shell ( But I'm sure this is the same concept as GCP/AWS)

### Related links

- [Requesting a change](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/)
-

### Use Cases

Users will not need to leave the browser to run documented code, simplifying the users ability to follow along 

### Visuals

Please see above link 

### Before submitting

- [X] I have read and followed the [change request guidelines](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/).
- [X] I have verified that [my idea is a change request and not a bug report](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#its-not-a-bug-its-a-feature).
- [X] I have ensured that, to the best knowledge, [my idea will benefit the entire community](https://squidfunk.github.io/mkdocs-material/contributing/requesting-a-change/#benefit-for-the-community).
- [X] I have included relevant links to [the documentation](https://squidfunk.github.io/mkdocs-material), related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions) to underline the need for my idea.

**Comment by squidfunk** [comment]
Thanks for suggesting. As I understand, some users are already doing that, but given that this is very much language- and less theme-dependent, it's definitely beyond the scope of Material for MkDocs. I'm converting this to a discussion, so others can share their implementations or provide directions and/or further ideas ☺️
